diff -uraN pylint-2.9.6/requirements_test_min.txt omv-pylint-2.9.6/requirements_test_min.txt
--- pylint-2.9.6/requirements_test_min.txt	2021-07-25 20:26:55.000000000 +0200
+++ omv-pylint-2.9.6/requirements_test_min.txt	2022-04-22 15:40:47.770585028 +0200
@@ -1,4 +1,4 @@
 -e .
-astroid==2.6.5  # Pinned to a specific version for tests
+astroid==2.9.3  # Pinned to a specific version for tests
 pytest~=6.2
 pytest-benchmark~=3.4
diff -uraN pylint-2.9.6/setup.cfg omv-pylint-2.9.6/setup.cfg
--- pylint-2.9.6/setup.cfg	2021-07-25 20:26:55.000000000 +0200
+++ omv-pylint-2.9.6/setup.cfg	2022-04-22 15:40:16.182796285 +0200
@@ -42,9 +42,9 @@
 [options]
 packages = find:
 install_requires =
-    astroid>=2.6.5,<2.7 # (You should also upgrade requirements_test_min.txt)
+    astroid>=2.9.3,<3.0 # (You should also upgrade requirements_test_min.txt)
     isort>=4.2.5,<6
-    mccabe>=0.6,<0.7
+    mccabe>=0.7,<0.8
     toml>=0.7.1
     colorama;sys_platform=="win32"
 python_requires = ~=3.6
